<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1144</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnMarket">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>РЫНОК</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPortfolio">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>ПОРТФЕЛЬ</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>1001</width>
      <height>601</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <widget class="QWidget" name="page_1">
        <widget class="QTableWidget" name="tableMarket">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>40</y>
           <width>791</width>
           <height>531</height>
          </rect>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>ТИКЕР</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>НАЗВАНИЕ</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>ОТРАСЛЬ</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>ЦЕНА</string>
          </property>
         </column>
        </widget>
        <widget class="QLabel" name="labelMarket">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>331</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>РЫНОК</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QPushButton" name="btnBuy">
         <property name="geometry">
          <rect>
           <x>800</x>
           <y>170</y>
           <width>181</width>
           <height>81</height>
          </rect>
         </property>
         <property name="text">
          <string>КУПИТЬ</string>
         </property>
        </widget>
        <widget class="QPushButton" name="addBtnMarket">
         <property name="geometry">
          <rect>
           <x>840</x>
           <y>40</y>
           <width>91</width>
           <height>71</height>
          </rect>
         </property>
         <property name="text">
          <string>ДОБАВИТЬ</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelMarkovitzMarket">
         <property name="geometry">
          <rect>
           <x>790</x>
           <y>10</y>
           <width>191</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>МАРКОВИЦ</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QPushButton" name="analBtnMarket">
         <property name="geometry">
          <rect>
           <x>800</x>
           <y>120</y>
           <width>181</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>АНАЛИЗ</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_2">
        <widget class="QTableWidget" name="tablePortfolio">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>40</y>
           <width>791</width>
           <height>531</height>
          </rect>
         </property>
         <column>
          <property name="text">
           <string>ТИКЕР</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>НАЗВАНИЕ</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>ОТРАСЛЬ</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>ЦЕНА</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>КОЛИЧЕСТВО</string>
          </property>
         </column>
        </widget>
        <widget class="QPushButton" name="addBtnPortf">
         <property name="geometry">
          <rect>
           <x>840</x>
           <y>40</y>
           <width>91</width>
           <height>71</height>
          </rect>
         </property>
         <property name="text">
          <string>ДОБАВИТЬ</string>
         </property>
        </widget>
        <widget class="QPushButton" name="analBtnPortf">
         <property name="geometry">
          <rect>
           <x>800</x>
           <y>120</y>
           <width>181</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>АНАЛИЗ</string>
         </property>
        </widget>
        <widget class="QPushButton" name="sellBtnPortf">
         <property name="geometry">
          <rect>
           <x>800</x>
           <y>170</y>
           <width>181</width>
           <height>81</height>
          </rect>
         </property>
         <property name="text">
          <string>ПРОДАТЬ</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelPortf">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>331</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>ПОРТФЕЛЬ</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="labelMarkovitz_2">
         <property name="geometry">
          <rect>
           <x>790</x>
           <y>10</y>
           <width>191</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>МАРКОВИЦ</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QPushButton" name="portfolioInfoButton">
         <property name="geometry">
          <rect>
           <x>810</x>
           <y>350</y>
           <width>151</width>
           <height>81</height>
          </rect>
         </property>
         <property name="text">
          <string>О ПОРТФЕЛЕ</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="tableAnalysis">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>80</y>
      <width>101</width>
      <height>191</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>ТИКЕР</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="removeButton">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>280</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>УБРАТЬ</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1144</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
